packages/app-content-pages/src/screens/Publications/Publications.spec.js

```diff
@@ -36,11 +36,6 @@ describe('Component > Publications Page', function () {
     )
   })
 
-  it('should have sidebar nav label', function () {
```

Comment:
I'm not totally sure why these tests fail, but this app is going to be removed in a few weeks, so okay to delete for now.